common_dwp_api_test/src/main/java/uk/gov/dhsc/htbhf/dwp/testhelper/v2/ChildDobGenerator.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.dhsc.htbhf.dwp.testhelper.v2;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static java.util.Collections.nCopies;
+
+public class ChildDobGenerator {
+
+    public static final int CHILDREN_UNDER_ONE_NINO_POSITION = 2;
```

Comment:
Might be worthwhile having either a quick JavaDoc for this class or state that these positions are zero based (so this is the 3rd character of the NINO) etc.